deploying/deploying.md

```diff
@@ -32,7 +32,7 @@ Deployment is a fancy term for getting your website on the web. After building o
 
 - [Video - What is Heroku (4 mins watch)](https://youtu.be/r5ZUQvl9BtE)
 - [Techtonica Slides: Deploying](https://docs.google.com/presentation/d/1Enwhd9hl1fn1-afMXJ6xvkJm5SDJpHjfQoA7s2znHpw/edit?usp=sharing)
-- [Deploy Node.JS Apps to Heroku (10 min Video)](https://youtu.be/AZNFox2CvBk)
+- [Deploy Node.JS Apps to Heroku (10 mins Video)](https://youtu.be/AZNFox2CvBk)
```

Comment:
I think it is better to add one more material on `how to run front-end and back-end on a same server`.

Reply:
I found this one: [https://www.twilio.com/blog/react-app-with-node-js-server-proxy]
Do tell if you want me to include this or you have a better resource

Reply:
I don't understand why - this outline is specifically about heroku.

Reply:
Ok, I won't add it.